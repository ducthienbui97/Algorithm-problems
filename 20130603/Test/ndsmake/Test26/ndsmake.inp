2785
360258054 -440883199 690180501 -270716174 -636079356 -91897806 -442065657 -840906007 565700021 -689435646 -55517304 246513570 -468782097 -558221249 -428102922 535173125 400747983 -700593560 -428524126 543424622 271527058 -282618829 86268104 28086422 260280674 615426817 -870803427 -197121241 464722073 168252836 426683578 -976213050 -642065117 742573891 -458756850 -454421930 388860365 365855360 321397720 974377278 -303195815 -491440870 -574844834 -306094885 -995867247 -851281742 -598634306 -344738989 862522382 -590819314 638545788 -322372838 283080598 -382779794 640646513 -583481456 -798890738 43500029 217620562 -253442616 854154782 611697507 790757340 -135777436 956973920 -125129609 -527130212 803151939 707786770 304296786 -600791459 -256466302 -91321327 -751485554 -378581187 279795214 565623762 -909067155 -381997135 -994137785 603702726 130177304 -55619687 -35639236 462319939 -309368792 -668563680 -507223177 613999947 -993815944 -29578623 395102995 392245942 -746625789 -920725187 -82700545 -651379477 -364951816 -89441059 101769641 -924405354 -129599503 -904635936 940312140 -971325709 -58619108 311236084 725815771 388811742 -969994529 646771940 -847906670 376661348 -914894587 -486828660 -133916308 175157665 -482772475 966996364 -561168926 830802767 647982789 -357089848 -304529649 -240647976 -97392204 780930984 -567816320 -306608924 -520622959 359952526 357608451 -268606461 879236335 276687172 -798758681 -948425411 563462591 9911786 963655065 -628126644 -254859797 388831117 -722182708 -214401792 -859731456 -447914929 -468326283 717133801 506082645 -794960485 -437098179 498735500 970582059 -490805905 -298691027 -961283138 284951271 98950382 430540204 103060521 894383462 -825953337 -959803027 -687399429 9026623 -544865222 615572848 485175176 720674471 703260055 468329555 503286408 -394204864 940095539 -975584114 -889127822 280676334 -319443589 -459898544 -349781168 397188576 -629981111 760262873 -957115653 -39055784 -488863558 -117691834 356255781 -248726754 633999283 -330901445 944361147 -366910456 -433329232 -932260995 504576748 350425988 598483909 -800954932 393088325 -799520408 -577375686 -993981017 76270599 527537347 -445797307 -598812660 861095617 528152691 848019507 -691776421 262175111 686894130 920036441 281881041 -164603976 -565622192 -42977551 -61107555 -909518444 613644115 -186173390 -876525763 814437519 -784548962 -442654070 -123097539 -856214338 5315356 150092868 69121895 -703944701 -475012466 -172535486 54806414 -83223327 -178434554 -307046117 -385854180 -247862670 -666619668 68652201 881372852 107704684 74648499 301303277 -91854681 724379687 717997165 -819953983 -447721839 -248215487 732738447 -164885817 -446916264 -932392664 -470822960 -977230884 -855626399 66601775 -929296321 -355120984 975060725 321341532 464724482 302829412 -805417031 560605499 -994759499 424751862 -141413286 898518707 318826353 766682663 -426542281 907947248 698979482 675838674 -772270546 182104316 918598532 -213532004 30357787 -288514393 812722954 193492591 -787458449 641287686 -729500372 412318824 -349626625 543331369 -933191394 -159094108 -519975899 291680528 950712263 -296508340 -439201946 343259360 712718532 -963574285 -350540453 980170224 515629264 -903008481 779309585 355811534 191848209 191446647 828760990 632952369 447455421 149187533 -798090074 846558223 -406942919 449778816 -530728127 679292835 -351088921 -728781565 -516262822 -539073303 -736272417 -719112305 280749827 -64883177 68147267 399268687 -28425313 -776677673 605170961 981044186 239366121 -233462648 988108909 881076630 -175449574 833312454 -826183070 -522870335 -263029293 430961693 885962944 433708579 703700127 -595858077 205934704 -407289610 731921563 249926834 -344684056 686175601 230083071 -170018119 -908132621 580289024 637704028 -997859771 103092425 637164504 -674883898 -17442660 -816079956 -372240853 -521828998 -451211045 159169021 -194633644 784004479 -386273665 -964729620 74630172 913609759 -526326011 499384131 -456179266 767010495 -980560031 930173199 -174608413 861743314 522593299 -36759557 741172326 83451273 486688940 -666174043 275549187 -196351812 -769084562 615337794 606074970 865580380 891184526 -407137427 846465334 -769736144 713375736 -432223329 261485287 -897112691 586134911 743260025 339593422 233460404 61353519 -115903824 -937988617 739056328 -907932226 328617877 -841044512 678200170 737636129 -285528000 -305864872 -600625933 -768227524 202575960 175805315 275554893 -214904446 18739962 -712536978 -669662502 -864646385 856515442 -123750959 674957397 40987950 67491669 122795436 618746787 735879775 -211432886 -244283706 648713856 -267447065 -875735366 687717688 658471561 72743644 296003943 496279796 645178876 820618221 794311607 -171015850 375501757 690811661 921055582 -943183089 -204551324 957527478 393673811 -66817197 404299851 -11260967 -289998748 -813712939 575086968 -135703145 123510631 -723659237 618013336 -722723017 -759676877 -506135284 -499691838 21102767 -658045103 -717960644 390722153 446000461 998296949 21502668 598325731 909628179 263172116 -274406257 -556002953 -584117859 -36716537 773806653 77100333 -932113203 -949795977 452857909 320608339 200521127 -682819724 -569608647 -216610714 544797236 962747001 981505929 -22038912 -661915247 -518085730 -722213893 481639052 -790411618 431848362 -77774210 3329255 -774855709 879986563 719311362 -139735571 666163817 -574636287 902964448 -817295835 -737594315 506042077 -176322329 -281315718 -110859166 738891720 457541124 370258596 929056474 -31910760 125853069 -646258778 35563952 248697245 734786602 -565288848 134263325 -893284590 263663712 -758048092 881843990 469797039 -540117061 -567116427 -784266362 59317340 421172376 708826284 87724589 498985471 -393482612 -965572155 -666725099 646897270 98964790 -804257108 -8012895 -927822053 646742479 -368536957 -30290026 725835649 189039676 396333986 -723342088 284070862 -55769467 495769185 298253267 -554944388 315759799 -466587908 -890565063 509549125 -755509238 709435952 -747865272 996216244 -943316111 953128055 266461782 360777150 356343674 491245389 -979487798 790135524 -805176214 -581092797 -294131252 -21008499 -368485386 -222047438 -96280181 -979492721 -741288804 -6326387 507658026 79384733 -710070064 -779852216 -133798151 -801135977 -453536235 233943338 304627834 -539407934 689861829 427241607 -724631609 465657329 547692470 -327284168 -303203932 -938521893 -256044184 328133040 527721523 -544495102 -404250488 -512199631 477106973 298524929 -537302221 -210989471 257370353 255961217 -705673310 692296114 -157893890 243346889 863926141 100316612 -580456648 498359043 729883102 -228153143 760202800 275313348 452266464 -977491696 159586128 626066980 -52797503 -455707749 646569127 622392466 97852405 669204172 827867474 154507149 11974998 7138731 533787638 197603323 -833347464 611380694 -512499863 221744937 -718948837 261671430 -78345360 -974678688 -908288326 -334161094 548833187 -853032928 -529045184 -839753107 880353788 -829328678 -406325911 -341062096 -498090479 785085927 -359872768 -318130555 669311268 -144648691 -380733565 -156555359 699904600 845437797 96023723 698354131 -607922808 769142486 579048591 333108009 -992372409 268437887 -98224841 780169523 -847197583 -704009906 -136082753 503768942 664190126 -681236955 589195907 -429191933 -772499619 -176791889 -912930822 -721345957 204400636 -987415835 258936186 -285107970 332824229 -12439277 319396925 -215562973 515691771 -162928530 826510463 363892176 -639289242 105890916 -9060077 719594299 -676509910 296540177 -716290370 347690890 592962576 440334655 -763052052 945063463 -609635881 -882537107 -896265364 -675878249 -367646126 -900403392 464126851 808175020 624980392 -715457203 464262092 -356304240 -31190190 488769483 -664221229 788123457 -126165386 373535819 -397613630 -759051180 -744337894 670557700 235481657 959636306 -382239028 -674409382 -415508502 989350078 -465346821 -513045044 -620767539 -526920436 -351172397 -430326564 705892666 250138762 73168061 490971073 117124478 291596272 387820710 -448523080 287837396 374038207 -256980331 -548561984 -802141003 -212872221 86370028 -215542824 502716806 -120181943 -352562623 -244734022 -422770975 499982272 -314941719 -738978026 -769272377 813070009 -261415871 -530312423 677243195 110787685 -433432019 257986565 971974653 -124140581 -955521515 -412709234 257796003 -635941933 -571389415 942305310 60492834 -973867779 893848929 219022787 -855260161 -323460170 -633023183 -714405622 780499292 -345824375 225789641 -125098216 656882736 -222208276 552461099 442520772 -657622257 434748181 -262844219 -927054210 -736228120 -258113201 -152067064 -561810026 -304233120 -482857207 -484594838 39446071 -31398115 80104544 838815324 9828392 -158728449 -272396959 -618491766 513930376 907682018 342618025 231187811 -698618516 -298008266 414973507 -340833193 -762629904 538284381 686011009 627299706 -805051249 -750984161 -720406928 106077222 -307501928 -699228066 668318852 -471459935 784153670 388536511 -953102888 695136580 228367326 -66550947 -437498953 396338840 735237956 274654951 987940870 175375856 924549436 453930041 599777241 -557087411 565735319 -354931421 -635186793 865353152 -93639802 -708222587 459239720 519743432 -846104762 -262071116 -764250181 562410223 -953864860 683466627 -569970818 -815195354 -437348417 -24061944 125460093 -407714700 161349063 130819805 852506954 -543745055 809917730 -69720624 -975027132 374604848 -355401881 -29512106 775737173 -336607303 114996581 18917173 -573999974 -759233560 -284670706 645684295 929757061 -882498915 -858361904 -822883142 259825980 504777430 -831949836 -74281807 316892201 -500476159 -34125416 -325299095 589271149 123616174 -833027688 -588725168 -395747975 -223093375 923467814 -101128811 -60026620 129166344 -99429962 349438621 -911896404 692363187 371490958 274750106 670795794 147882572 135040181 -547545202 -140407006 -512621291 290730250 -151385345 334246512 733159037 467904368 -546256514 95370791 196832051 634443930 -658382698 708259894 -234318032 -168989404 260468319 653218564 619904976 -664411923 -414372403 312658695 320900580 -384599045 673092612 -138864237 -645498662 -377591318 -468165164 285537748 -646114877 -923915987 -147178798 -577076521 337704191 337800305 -168570860 422679746 -596808568 -954896350 630058120 -722348252 -118036986 -619943695 324126311 186988150 857651464 -817640064 -31373789 556337936 -919242997 374472715 100654494 -689901811 -67725844 905769034 522129168 -164265708 521273465 685732328 -579480296 474816162 -359826266 785473907 453288189 -878544883 206597190 400858041 195612851 -632743019 213380619 -923483817 492558996 706590576 -951497194 183873544 -477885962 -165472828 83406787 764119183 -913492971 -186472386 -857222844 346555135 601478325 -78952721 -260535025 839142710 -857408896 -889168026 -807622140 756509435 289176716 -987145671 707018685 -894168019 -232199447 373252243 948102268 405968176 -897652589 -651652092 -564133111 -392672042 -15507795 -102900253 86684815 -962351551 -549851591 396200964 -586116399 152430809 -744016853 283891695 294985834 736839169 -513303312 -200041124 528381344 -612946854 -373187457 143328327 863079031 -395520295 -338368493 -769892164 520215774 576106037 -76549376 -131518185 -764622797 -903882444 -261951325 -247325769 298077237 -911647522 999681275 -470021454 820289722 -754527112 170258663 247714143 -105083763 -450404005 587597317 -588172985 -548716159 -299831987 708507023 392911121 -687991146 226762654 -842240063 512655806 354251785 -805405943 -122812558 260135485 427996311 -727332196 371002404 119760168 960519795 -250646877 -159890619 -272043399 639197199 904575666 -616056063 351781467 95621002 477884865 225559326 904361654 -751852943 450307397 -436175803 405542340 200820077 -460433318 986372282 819586703 891559916 -242022856 984097314 -354022783 -84439982 921559162 -84009142 -907596506 -306713866 -24486726 811952064 -776321717 655727871 -787115319 665722495 -983957806 -953709721 -633699192 -451910206 -668551402 982572065 516213094 930545145 -762210827 -787376297 562575222 17277401 -209982082 -3334452 -184826837 -747384434 18708833 -169904461 -145367284 -955236851 755576195 -455295263 -915448822 539604465 922032500 -250889621 172513725 206963409 -640024896 246410806 -669592803 722466033 944473806 -780315732 -648339425 -624180383 860029451 -141040384 -885572950 888456930 706964879 -902134697 -57749936 -270692601 66895458 943844808 595045249 -218357920 853174480 -932382302 509617195 153961616 831734080 -277498584 886185789 -123007987 -817799051 888234804 65983052 816436866 -371122090 -880202771 423044200 776510980 773174209 -979104870 735676397 -884603057 324978422 -273438509 -481310172 -371193896 89005617 802491332 -204764050 10191698 786461193 530009221 -402380498 -162463923 715224936 284529799 235781459 595963640 687746343 -418216663 -941424507 671758599 -218093242 80885098 956052971 382515632 -698466363 584128697 426254231 333554383 66747681 -292994839 419462799 -232834007 -651489682 421677433 409052621 -970239730 634484911 -915774494 -288792938 980149259 -516335333 -17154005 -534482738 -573819625 -682491430 307525136 -212695315 699769629 107076409 -57149482 -571061633 775417987 502047034 -362729455 -246771933 -817340673 -24110245 193131023 -524259565 -704954479 398313188 145720177 709811494 803689044 138068243 102359441 786241089 -358500909 -461922090 141879442 378863202 -223462839 572830475 15333367 -883522944 -529211033 -288262981 -338327038 -894824420 32188145 -422861497 901033089 668467301 652546353 -533147584 -385612392 869459413 563019652 705995850 114587242 -862890538 125750241 798269196 475568434 -962894191 -276965592 933640007 -584474338 -415820297 146774207 525646761 687953314 795028241 238874988 -86890119 -936011147 800246087 -336544997 -488667334 -33652780 392733708 -36755083 -415232621 726654641 -156994624 91846762 282400036 295402540 -88894037 450075202 -102860294 -577027076 379945594 294064861 -192462170 122538620 594085141 448225018 521641436 -79009007 -82807745 -289384353 -55363464 -891029180 -629679151 740591173 -565080663 13983517 241513865 -167637144 535509338 185004860 -855212531 830356529 -269339548 -747312074 337395005 700865176 507657288 429222306 568534217 -568364138 -485355368 -936162928 799181598 61994792 -492895160 -838270657 -1462086 -935309523 -516916218 485316554 969739626 82571293 897508821 -657654548 602563102 719524143 511595619 -241985009 930882116 -603045447 268474618 -898730633 920762213 460764801 410068187 293248733 70841111 556393213 -331314610 -743318349 -280993771 678124907 -526203829 822733701 -470856964 901755389 100798142 -572190341 -473736032 -862193870 446347643 -895408133 5308350 -830317205 416509453 503763629 479759965 445190420 767678603 -687220739 919995020 253119266 -555823295 -579797882 463293261 45642440 749541451 -829338017 506754930 154272896 477980535 -865002928 702950000 177501643 725860802 -604040958 141768936 106200396 434691213 -33418665 741521288 -151706381 -479194279 -859141249 332321898 -658787480 1649275 138915182 -192267977 763236466 -55428258 -750155414 -899786143 -530427272 738350400 894832289 451451145 -520878413 690441729 -69130220 962346516 12416833 459588563 161951234 -734343960 -192020392 -675721346 916048741 -214136892 -430225769 69751118 -132220523 844537786 745947256 -790404454 -890383349 -303420855 -823847439 777332289 -344025445 -135630629 717180898 403682740 -610534313 823547483 -805690776 -293377525 -536527324 -970258884 -234572560 -586902190 727161276 -987535539 302539529 754892383 731176872 -438637435 92724135 731672405 838282398 582114388 -617698540 468068935 -616522125 -510625003 -430032524 561038836 -500752706 902146720 84142067 699765936 19135755 58241277 -141912435 -567093404 -210718023 473167923 -504595335 -174514516 -772421007 621656382 -730145485 -290300898 -435780285 651863070 -562461270 788641548 109547761 136695069 -223873428 -721348920 747814493 400199104 657052552 815716877 -166924836 -144887267 730888925 432305121 -46124324 -59658286 329807837 -586689255 676231683 -812902616 -820942809 -578094397 -946668035 -116948547 -715501987 432551614 -76338118 252501339 395841545 906663775 146096817 995235023 -470773984 -787527229 956252406 632001502 164368549 -392439604 98325669 -102637626 134949739 -774954567 418339576 -332140948 -645029641 -507594655 813345144 -443578841 -377838973 940840204 -327782447 999526439 -402578401 695346366 897902467 -506527757 -279675327 -393048029 -182661409 97942782 373630009 -920594762 -269459005 905930039 -589662237 44473930 674611732 407163936 828537125 23725761 -465629080 164947610 -301554870 -643409130 571852783 -585249957 472468989 -768547652 562331175 55230367 -233778630 -522576697 -347869244 943119737 332318403 839960032 -937102755 762634981 -612327537 964628506 -520424599 391209308 -91805037 -603669931 -238759612 -823059149 -420072510 703868398 986274564 -477864452 -403158897 644431749 -628735134 -248818215 499359749 731612619 -390509079 -625379844 -628514447 517613034 804623387 565733922 -366767277 353155761 -924511825 -116521106 350719393 -718866176 627355751 692958719 -193608594 -487525257 -698732346 -846095862 -157864269 -689507842 -82755377 -240550892 69337367 429931403 -563913124 -213668784 -949986049 -440790924 -436741828 549910019 -3051664 792312864 522297165 -145027051 -227922586 254109635 -323373661 -598075943 -21402296 424849683 475372249 -650245691 -717700825 -731441327 -52453367 628021579 -704861160 -865830590 773503628 881781645 -841018401 545649703 496048005 835615967 -243528431 892755987 326523587 603828506 435117237 -655922650 96638261 96564065 94897087 751613355 -817139939 275663872 730682234 310429741 989743628 240789229 29525050 -990119298 424910265 181342298 411915705 -196227748 -778154696 -398287648 446794213 886854204 74976842 -383324096 -684725019 324957581 372239974 -334619972 -986487030 338442847 225448835 162179867 582284196 804172079 -410878077 -815784246 73526145 -234927648 295591996 -800132580 656555267 -272821557 259397155 981611186 -732181539 213227245 -363414633 -551512586 -186002838 443034943 -332622136 841137834 784706339 189562946 547632413 87336153 -277115910 362342426 -929976540 -750255698 548978642 874103731 -89820665 165291565 -309628099 -838250963 -583784405 -298039511 -111525315 810296684 618574189 -356922790 -487985728 682790534 993528598 -734289962 -204771153 768836089 765228922 -277897589 -245280662 -103982459 -405477529 -372325542 -321145577 878477390 585235245 -207530643 -210516398 -881957662 925807134 586500828 495528276 452111715 -772545771 598698886 584078710 -789395850 21341827 -742493360 498562628 -620161257 764335757 -411495110 339904510 -386059510 510863994 -19782003 -424455200 -417421063 181528487 -184028242 -290144804 196268211 346720782 -39296299 -229639493 621020490 601085266 34085175 -586395393 -584876369 -612138073 684731374 767547007 -669147843 757577425 -987924362 -166011990 -510926634 998672956 751174237 755992812 745600924 539194408 -806469242 905097257 944029257 413419521 213437209 -138349466 -530994132 175819724 809732487 -912079679 -583732546 -739572667 782560999 -896434829 -366861869 -722533936 -256676853 -44498836 -55641926 -726425775 574711185 -415778247 -632008609 -688045019 673458581 -550286908 547030469 239532433 538786584 54328095 -651513861 -130718522 -19339945 562703725 531972287 -963397673 -157715911 -813896335 17878334 -695961515 727848417 -165187919 214294775 -320219768 -229392028 -902451898 488430781 961210444 -308334956 707994179 -660929573 -382282105 -535652613 123147456 184316995 496930621 119315564 -113986744 437774933 -708371673 -417411180 285279668 -387048559 76915660 134887472 -733516587 413364710 -164238206 807428348 -571024563 472770092 292268406 -670462323 -246693597 771367713 119268678 388644136 -781202554 275101618 -770080430 -844514562 -560414617 -235282619 -330638420 -810014775 480725368 -757518757 -88897674 -860675685 -54886911 798337468 448484515 474188159 833929697 487943415 -253250262 -12803486 -80790434 -953471667 78148342 720665864 556255782 -721151105 -172484510 -93489242 -118626611 811582815 -174214746 -467612454 -917083672 926437763 834309439 295644531 -749026097 277736485 702448182 -430481635 253368880 -778114862 -737573652 46182700 -397389359 -365057947 304139158 798465275 -338133206 460059110 257515130 507593363 826874356 399861966 -759275036 -397521493 865874228 860790504 433916170 568339768 727749591 428435569 -613410167 -106094523 170170299 -369243449 -525186360 -862143061 -511377433 237693873 439917177 -746390160 288281564 325806917 -136520898 108805562 603855363 698692218 -530539416 -321233554 -271546555 -262971317 -126688806 289056362 -322808391 208248707 -171497716 282976603 947688568 502976299 538071083 257396027 874472077 553748366 -14005410 700372074 206022409 -943217178 223298596 -27559576 -506005453 -271406939 452029145 15897901 681411548 -561166751 204763736 469410085 166783695 -305968804 539100601 565962444 565715303 -670344785 -804349893 492441313 -341604548 -754253805 -683142851 -728421814 625575925 -340014260 98301543 -480498044 869480257 69018163 -717564442 129007556 -391320482 -589770028 -294010193 -741708323 121232238 -741911115 -362623053 821552087 449101556 -121502947 -806691972 -439161600 -60212624 116309031 902108438 -999659192 998670018 411999309 831591666 358304733 -302989393 -197996289 -230622880 251226086 -540900810 828841299 -513249788 873513158 -187867547 863143162 -218138923 -889055831 -71330945 -955898000 -940864619 114340048 580336691 370136094 -394332853 -143513766 -77484177 198917076 -682756975 883042561 403434629 -439504237 -98007987 457735220 -553762390 538314742 -731121477 368216629 673523121 883156963 49645980 114759267 521846334 391177773 -847918405 837922093 -468822437 -404510131 694922863 301695337 -342787635 -829932208 -283775427 948864340 -932039807 -286323618 934730023 -906133211 621103591 -112543026 891177298 -496751148 846853995 -355603981 -71354793 892167657 938312166 941897245 398816757 341736151 -139894636 -609439791 -479443253 130324460 -905251268 444053259 388909240 -76299444 -581911786 -942849086 592158672 -965499264 682167263 -729744011 -258178205 -636175873 -754026538 -896227085 54452879 -632126636 330167451 -72266861 -548896785 -425505810 -20369869 255657994 -780544399 -716162428 -892955352 -633622804 404830534 -101361373 391703757 365796546 -559759118 100118997 -163431521 -729800764 -7293053 943213552 911351751 341589811 -821471778 -491949064 872262077 122969685 -62274508 -847751119 -277938426 -889268978 724791188 -568089658 -129161442 722679832 872408731 530080399 -2063248 -377378081 -911175866 916526979 -792814719 383202556 -636835864 -467169350 133163565 328709451 423961911 140024860 413970698 -421155669 -686481536 153189292 -981880599 299481681 -710057877 -709099909 -990952503 -199571655 -733603606 770387471 -745971061 367086009 -689862544 543871286 -636315749 89536451 311140879 246231659 360767084 894167191 -880840499 -923817143 583625760 606188944 704239256 163212160 620035613 -590223114 -42643809 -736862956 -426987661 -202091532 -303610658 -341715536 486168618 -922578803 -926915907 66503588 627462069 -896649188 876200917 -726372227 567261317 -43944750 53646144 566950579 -958611853 -59438013 403530726 511862644 268230318 -452269157 -39128591 504089461 -417372119 -64531307 49933956 -315725228 712722465 628596798 33141770 514886629 -839441295 -548367598 -455847641 -258773573 -502137924 -345063109 251738106 933848380 -992443317 731755659 154577319 -258637389 -773221631 -237436935 -973270598 779696078 228872394 730213867 -763963298 -32383795 282950462 271454687 -808270545 -421245732 464603070 -329745331 404813308 687331538 412529026 -324239150 -120823634 367486557 -441037855 -316950494 294695869 298084645 998706196 838075407 599681478 -826182390 528295274 665284668 -833789609 800348956 894971682 671886314 -299571367 97899715 -52982296 -546763026 -822493670 47546111 886160111 -13091901 711019255 331548618 -509959513 273715388 356627619 -851865702 -185666277 -79709614 -96904170 -473407229 -321003029 -953558885 781498900 -454803336 587749857 849398236 -846181544 -287394254 -442200015 -306951115 -611003043 -309013463 430314834 -98744198 863432138 -203613633 -724138637 419018787 -416854408 -145474948 -181377892 -768362407 844111905 790798196 -804429214 -679117776 -793508442 -758073659 424861510 -315297804 -697937412 -899800643 73084331 809303964 -239045963 615517720 -477648781 700944988 -722519769 -684910362 -271623883 -105222826 -585024117 14649372 650792816 68857044 -446008608 -333500577 -315967423 925751001 957790328 716958808 -492442588 -9852703 673609213 880191556 -691017873 -553132934 -692965704 -271592810 692064958 438991126 576760623 -334059784 157854312 630934419 780851944 610048223 -387978477 -21180340 -620388663 598507106 -275294041 507492118 -545948978 447901738 59494988 431464576 -340171380 215755313 -878443818 -16359367 -288043606 715289451 111267720 -503304900 -969935213 912185814 -2356321 -946477646 987918871 -627957251 953499555 -482588604 -249433651 -121051591 -9800825 -385191545 -964609726 -863225822 -936546743 -104503071 911551443 282744474 -829133438 203804198 -759199672 432645494 316794234 507930705 796386669 276183027 984666683 -827167286 731863644 -391193756 537697412 634102004 -860029483 -412268621 -302161022 -190563726 625166297 -122298454 822961557 518118714 997792259 232256443 18004723 -864404652 368666722 544420764 498518502 -199311147 503825552 -68379119 -375386700 -2860358 60681801 -139704650 434720241 96786606 751928889 -568372713 -651154030 188996706 -347120347 684032811 551469611 464088734 529732669 -572786583 374002744 -674399960 -740068508 -637598201 -886314062 21649610 562409483 -855216080 -388562142 759889980 909692645 -60561018 453082886 685579187 893064557 95258917 58210705 -862556890 -868821126 248951740 -830679642 -293501478 -873457758 890770689 -96336866 -281510698 60320696 913425805 -501579017 275340843 41507545 -811932027 283950558 723243170 394949745 927761874 -34905699 -91660466 -488549339 -721608963 205690918 715248226 -522759504 -94269287 221627055 122026924 641741262 71810641 939750390 -871853902 896685392 720701834 -348156762 325340263 -568837296 711682740 -483427374 298857098 -770009806 372426142 -259101028 427361690 327291389 589293692 806117115 -376910771 -340712044 411977888 -650318004 654304870 830752446 685753842 54775894 95409918 49262761 178501445 -245242633 -619887924 94361245 37519986 -45076461 -95200931 603661075 -727814815 582725798 216670940 -957256571 527339587 -190338348 -1475183 -361497154 -62849224 51053732 320506535 -276912938 -417201807 203979540 -63789445 -148924937 -566374959 668457319 -625150679 -215840643 794368488 800564548 -515950325 -940989210 -32647690 -47043842 -818194738 -273014810 793993118 817351663 -439959246 -982109033 -745605847 378310909 -796219810 53203827 -787261219 -502855562 292515000 -670888391 846308465 598033217 -559698638 -952367125 -506686540 126590853 -585824869 -852757719 158075659 -513342938 965766098 55633374 -886763324 -107624194 831541722 -702898192 809913439 332093056 -306571241 844220857 73787090 7540028 -288231515 906028406 670056833 140789844 -164678337 -905639535 -631217380 47047748 323256970 72794431 -779924710 -720095510 803618945 366832253 -175507748 929618525 -556382693 452774642 178336879 30707236 -674097622 742640256 490623402 24381907 159569035 -865841868 -136359146 400758664 698576736 -770435130 686885700 170352952 654948096 421542463 590941643 -929988315 -27735426 455164597 788633565 190932844 -572947755 -643786190 -909301626 242894522 -942266707 -857502285 -960945536 969103903 790712524 303473996 915567029 -686426629 -876309732 -648279191 563400274 -420579820 886177876 -93723089 -176367741 832355018 144467526 17933568 446619601 747243362 -523459899 -128490467 -977804010 -176868482 -796451535 47229031 -942944743 -714915411 918228641 334746942 20686794 -107901665 241558187 200662666 857202801 -860906798 -588654376 -744424328 -516167769 99204749 -869389924 495245797 199003959 -448200208 -923977323 297419955 838009410 810183930 598977358 -854582818 817327482 -906217536 -820673529 -652726386 466148766 -617516267 -74037750 879518931 815769881 891804551 -299047621 -11562533 -322282768 815182951 492982592 717630808 -579240478 280776506 -609572979 43229036 -872851086 409768384 44447122 46285194 -322604038 790425268 762900206 370221843 844992514 207184305 -786676270 -554545206 -195421579 -765506918 -237506120 -906771549 505141445 612578912 -416996494 302495532 175967924 -368001040 -673453907 -102971003 -28688977 282956822 36913096 -998068546 451100530 630974496 383339571 388899182 -409413070 -847808341 72692453 -646831722 -407304525 110525133 391281118 -709966720 -741119775 564548145 725044196 842811764 -360841994 956135773 -939315751 -833750818 -131737504 -568283663 993455982 -16020450 139974337 452748924 -299518732 -931330067 -836774640 -253129733 34835539 -75918489 -517653283 533336303 521871123 -267881031 26591665 158965786 -409259112 53216359 -774155629 454573270 939904875 444656698 547905959 -503820527 -982127438 -870785339 3523570 181875944 -510798654 605965971
