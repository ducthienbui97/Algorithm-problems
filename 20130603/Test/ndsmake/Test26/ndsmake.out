1384720778781
